EventFilter/CTPPSRawToDigi/plugins/CTPPSTotemDigiToRaw.h

```diff
@@ -0,0 +1,53 @@
+#ifndef CTPPSTotemDigiToRaw_H
```

Comment:
.h file is not really needed for producers and the content can be merged to the corresponding .cc file

Please use an include guard matching the full file name subsystem_package_fileName_h